server/src/main/java/org/apache/druid/server/log/FilteredRequestLoggerProvider.java

```diff
@@ -63,12 +68,15 @@ public RequestLogger get()
     private final RequestLogger logger;
     private final long queryTimeThresholdMs;
     private final long sqlQueryTimeThresholdMs;
+    private final boolean logSegmentMetadataQueries;
 
-    public FilteredRequestLogger(RequestLogger logger, long queryTimeThresholdMs, long sqlQueryTimeThresholdMs)
+    public FilteredRequestLogger(RequestLogger logger, long queryTimeThresholdMs, long sqlQueryTimeThresholdMs,
```

Comment:
According to Druid style, should chop down the list of params:

```suggestion
    public FilteredRequestLogger(
        RequestLogger logger,
        long queryTimeThresholdMs,
        long sqlQueryTimeThresholdMs,
        boolean logSegmentMetadataQueries
    )
    {
```

Reply:
You might want to look into using spotless as a pre-push hook so that you don't have to catch styling issues in code review. It will automatically re-format the code according to a provided style guide
https://github.com/diffplug/spotless/tree/master/plugin-maven#applying-to-java-source

(long time lurker) Sorry if this is a redundant suggestion and you already have another tool that does similar stuff